lighthouse-core/audits/is-on-https.js

```diff
@@ -41,9 +43,25 @@ class HTTPS extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
+    const networkRecords = artifacts.networkRecords[Audit.DEFAULT_PASS];
+    const insecureRecords = networkRecords
+        .filter(record => record.scheme === 'http' && record.domain !== 'localhost')
```

Comment:
i'd prefer to whitelist what we consider secure, rather than building up a blacklist.  

check out https://www.chromium.org/Home/chromium-security/security-faq#TOC-Which-origins-are-secure-
i think we only want to do https/wss and localhost for now.